Tests/Indicators/SharpeRatioTests.cs

```diff
@@ -18,63 +18,66 @@
 using QuantConnect.Indicators;
 using System;
 
-namespace QuantConnect.Tests.Indicators 
+namespace QuantConnect.Tests.Indicators
 {
-	[TestFixture]
-	public class SharpeRatioTests : CommonIndicatorTests<IndicatorDataPoint>
-    {   
-    	protected override IndicatorBase<IndicatorDataPoint> CreateIndicator()
-    	{
+    [TestFixture]
+    public class SharpeRatioTests : CommonIndicatorTests<IndicatorDataPoint>
+    {
+        protected override IndicatorBase<IndicatorDataPoint> CreateIndicator()
+        {
             return new SharpeRatio("SR", 10);
-    	}
+        }
 
         protected override string TestFileName => "spy_sr.txt";
 
         protected override string TestColumnName => "SR_10";
 
         [Test]
-        public void TestTradeBarsWithSameValue() 
+        public void TestTradeBarsWithSameValue()
         {
-    		// With the value not changing, the indicator should return default value 0m.
-    		var sr = new SharpeRatio("SR", 10);
-    		
-    		// push the value 100000 into the indicator 20 times (sharpeRatioPeriod + movingAveragePeriod)
-    		for(int i = 0; i < 20; i++) {
-    			IndicatorDataPoint point = new IndicatorDataPoint(new DateTime(), 100000m);
-    			sr.Update(point);
-    		}
-    		
-    		Assert.AreEqual(sr.Current.Value, 0m);
+            // With the value not changing, the indicator should return default value 0m.
+            var sharpeRatio = new SharpeRatio("SR", 10);
+            var time = DateTime.Now;
```

Comment:
Same comment about `DateTime.Now`, think it's clearer and probably faster if just using a reference time